include/glow/Base/Tensor.h

```diff
@@ -121,6 +121,9 @@ class Tensor final {
   /// \returns the number of elements in the tensor.
   size_t size() const { return type_.size(); }
 
+  /// \returns the number of bytes required to store the tensor.
+  uint64_t sizeInBytes() const { return size() * type_.getElementSize(); }
```

Comment:
Why not  just `type_.getSizeInBytes()` as a body of this function? Also, may be call it `getSizeInBytes` similar to how it is called for types?